test/PizzeriaTest.cpp

```diff
@@ -1,22 +1,24 @@
 #include <gtest/gtest.h>
+
 #include <string>
-#include "mocks/PizzaMock.hpp"
-#include "Pizzeria.hpp"
-#include "Margherita.hpp"
+
 #include "Funghi.hpp"
+#include "Margherita.hpp"
+#include "Pizzeria.hpp"
+#include "mocks/DummyTimer.hpp"
```

Comment:
Since we dont use DummyTimer you dont need to include it :)

Reply:
Right, I forgot about it, thanks :) I will change it some time ;)